src/observers.ts

```diff
@@ -62,12 +49,15 @@ export class EventObserver {
 
   private throttleManager = new ThrottleManager();
 
-  constructor(win: Window, doc: Document, onEmit: EmitHandler) {
+  constructor(win: Window, doc: Document) {
```

Comment:
we still need doc?

Reply:
We still need document element for scroll event,but we can use `window.document` to replace it.